src/templates/sitemap.xml

```diff
@@ -11,11 +11,6 @@
         <lastmod>2019-11-23</lastmod>
     </url>
 
-    <url>
```

Comment:
Looks like the entry was removed entirely?

Reply:
No it wasn't. It was just moved to line 125 as it lists them in order and removing year changed the order.